lib/pages/chat/chat.dart

```diff
@@ -251,6 +252,45 @@ class ChatController extends State<Chat> with ImagePickerMixin, SendFilesMixin {
     _loadDraft();
     listenToSelectionInImagePicker();
     super.initState();
+    SchedulerBinding.instance.addPostFrameCallback((_) async {
+      _askToAcceptInvite();
+    });
+  }
+
+  void _askToAcceptInvite() async {
+    if (room!.membership == Membership.invite) {
```

Comment:
Clean code Return early
```suggestion
    if (room!.membership != Membership.invite) return;
```